srcpkgs/seahorse/patches/allow-building-with-gpg-2.4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/meson.build b/meson.build
+index e29b532236e5543682832725d79dea0c7e49f8bf..23d0b54f567c112249fc72ddb91ac8e40218c60b 100644
+--- a/meson.build
++++ b/meson.build
+@@ -26,7 +26,7 @@ endif
+ # Dependencies
+ min_glib_version = '2.66'
+ min_gcr_version = '3.38'
+-accepted_gpg_versions = [ '2.2.0', '2.3.0' ]
++accepted_gpg_versions = [ '2.2.0', '2.3.0', '2.4.0' ]
```

Comment:
Could you open a separate PR for this fix?

Reply:
Will do

Reply:
#42055